supabase/sql/z_rls_initplan_optimization.sql

```
@@ -0,0 +1,132 @@
+-- recreated here too. The `z_` filename prefix makes this file sort LAST, so
+-- when the SQL suite is re-applied in filename order it runs after the source
+-- files and wins. If your runner does NOT apply in filename order, apply this
+-- file last manually.
+--
+-- Scope: gear, students, gear_logs, profiles, admin_audit_logs (the 5 hottest
+-- of the 38 flagged policies). Lower-traffic tables remain for a follow-up.
+
+-- ============================ gear ============================
+drop policy if exists gear_read_tenant on public.gear;
+create policy gear_read_tenant on public.gear for select to public
+  using (tenant_id = (select profiles.tenant_id from profiles where profiles.id = (select auth.uid())));
```

SQLFluff (4.2.2)

[error] The 'where' keyword should always start a new line.

(LT14)